src/main/java/httpServer/ListenerAndResponder.java

```diff
@@ -10,8 +11,10 @@
 
 import java.io.IOException;
 import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+import java.util.Locale;
 
-import static httpServer.http.Constants.CRLF;
+import static httpServer.http.Constants.crlf;
```

Comment:
Constants are typically in all upper caps; sorry that I forgot to mention before!